feat: updated miscellaneous documentation (#51)
godot-escoria/escoria-docs: getting_started/step_by_step/6_handling_inventory.rst

    say graham "Why should I throw it at the blackboard? I'm not angry!"

Try it out!

Add in some game logic
----------------------

> Lets make the game a little more interesting by adding some logic to change

Reviewer: ```suggestion
Let's make the game a little more interesting by adding some logic to change
```

how the game reacts to events. From your previous lesson, the
"blackboard.esc" file should contain the following.

.. code-block::

    :look
    say graham "That's good to hear. I'm thirsty."

> Lets change how Graham responds based on whether he's picked up the drink or
> not. To do this we need a condition statement, where we look at whether
> something is in a particular state, and act accordingly.

Reviewer: ```suggestion
Let's now change how Graham responds based on whether he's picked up the drink.
To do this, we need a condition statement where we look at whether
something is in a particular state and act accordingly.
```
